### PR TITLE
create a new branch Books module

### PR DESCRIPTION
**Pull request by Mariagirbu**
New branch MG

### CODE REVIEW

File: src/test/resources/features/B28G31-151_MG.feature [hunk 1]
```diff
@@ -0,0 +1,20 @@
+
+Feature: Books module
+  As a librarian, I should be able to add new book into library
+
```
Comment (farukgoc) [inline src/test/resources/features/B28G31-151_MG.feature]: @ui @db

**Review by nurseda916 — APPROVED** [review]

**Review by farukgoc — CHANGES_REQUESTED** [review]
There is no implementation on your step definitions. Why?

**Review by farukgoc — APPROVED** [review]
Good job Maria

**Review by untuzina — APPROVED** [review]

**Review by Alexa3462 — APPROVED** [review]
Looks good!

**Pull request merged**